posts/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+#from users.models import User
+
+
+class Post(models.Model):
+	post_text  = models.CharField(max_length = 5000)
+	post_title = models.CharField(max_length = 100)
+	pub_date= models.DateTimeField('date field')
+#	post_creator = models.ForeignKey(User , on_delete= models.CASCADE)
```

Comment:
I think user model is already created. You can remove the comment and test it with that. Also, I think `post_creator` could be replaced with `author`

Reply:
First we were thinking it as one user can create many posts so a ForeighKey was created .
But now Divyanshu has said it will be a ManyToManyField between user and post and I just saw that in the User model the field has already been included . So I think that we can remove this commented line.